python/cudf/cudf/tests/test_s3.py

```diff
@@ -18,10 +19,20 @@
 import cudf
 from cudf.testing._utils import assert_eq
 
-moto = pytest.importorskip("moto", minversion="1.3.14")
+moto = pytest.importorskip("moto", minversion="3.1.6")
 boto3 = pytest.importorskip("boto3")
 requests = pytest.importorskip("requests")
 s3fs = pytest.importorskip("s3fs")
+flask = pytest.importorskip("flask")
+flask_cors = pytest.importorskip("flask_cors")
```

Comment:
Why are these imported but not used?

Do we really need `flask` or can we write tests that use Python's built-in HTTP server instead? I'd really like to avoid the `flask` dependency.

Reply:
Removed, comment above.